TransferMethodRepository/Sources/TransferMethodConfigurationRepository.swift

```diff
@@ -40,6 +40,17 @@ public protocol TransferMethodConfigurationRepository {
     /// - Parameter completion: the callback handler of responses from the Hyperwallet platform
     func getKeys(
         completion: @escaping (Result<HyperwalletTransferMethodConfigurationKey?, HyperwalletErrorType>) -> Void)
+    
+    ///  Gets the transfer method types based on the parameters
```

Comment:
add fee and processing time as well in the comments

Reply:
Updated comments